src/pages/proposals/index.tsx

```diff
@@ -96,6 +109,14 @@ const Proposals: FunctionComponent = () => {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, []);
 
+	if (!featureFlags) {
+		return <Loader />;
+	}
+
+	if (featureFlags.find(flag => flag.slug === 'new-proposals')) {
```

Comment:
You don't have to do this, you can simply add a property to the type `FeatureFlags` on the `stores/feature-flags/index.ts` file such as 
```
export type FeatureFlags = {
	assetsPage?: boolean;
	nftIdPage?: boolean;
	ibcTransferPage?: boolean;
        newProposals?: boolean;
};
```

Then access it like this `featureFlagStore.featureFlags.newProposals`